src/net.h

```diff
@@ -60,6 +60,8 @@ static const bool DEFAULT_UPNP = false;
 static const size_t MAPASKFOR_MAX_SZ = MAX_INV_SZ;
 /** The maximum number of peer connections to maintain. */
 static const unsigned int DEFAULT_MAX_PEER_CONNECTIONS = 125;
+/** The maximum TCP send or receive buffer size */
+static const unsigned int TCP_BUFFER_SIZE = 65535;   
```

Comment:
I suppose that's true, it is a MAX user defined setting, since the OS
can reduce the Buffer size if/when it needs to cut back on memory.  And
it is currently related
only to Windows. 

I'll update it and push.

9:01 AM, MarcoFalke wrote:
>
> In src/net.h
> <https://github.com/bitcoin/bitcoin/pull/6693#discussion_r39871260>:
>
> > @@ -60,6 +60,8 @@ static const bool DEFAULT_UPNP = false;
> >  static const size_t MAPASKFOR_MAX_SZ = MAX_INV_SZ;
> >  /** The maximum number of peer connections to maintain. */
> >  static const unsigned int DEFAULT_MAX_PEER_CONNECTIONS = 125;
> > +/** The maximum TCP send or receive buffer size */
> > +static const unsigned int TCP_BUFFER_SIZE = 65535;   
>
> Shouldn't this be named |MAX_WINDOWS_TCP_BUFFER_SIZE|?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bitcoin/bitcoin/pull/6693/files#r39871260>.
>